qt/scientific_interfaces/ISISReflectometry/Reduction/Group.cpp

```diff
@@ -32,23 +32,35 @@ std::string const &Group::name() const { return m_name; }
  */
 bool Group::hasPostprocessing() const { return m_rows.size() > 1; }
 
-/** Returns true if the group requires (and is ready for) postprocessing, i.e.
- * if it has multiple rows that have be processed successfully and are ready to
- * be stitched, but have not been stitched yet.
- * @param reprocessFailed : if true, groups that have failed will be
- * reprocessed;
- * otherwise, they will be ignored
+/** Returns true if the group requires processing; that is if its rows require
+ * processing (note the 'processing' here means reduction, not postprocessing)
  */
 bool Group::requiresProcessing(bool reprocessFailed) const {
```

Comment:
```suggestion
bool Group::requiresProcessing(const bool reprocessFailed) const {
```

Reply:
All lower passed values to subsequent functions could become const.

Reply:
I don't think our standards specify whether to use `const` when passing by value (there are pros and cons so I'm ambivalent). We have a mixture of styles in Mantid, so I've just gone with the local style here - it's not so obvious from this file but all the other files in the reflectometry GUI omit it.

Reply:
That's fair. I tend to take the approach of "Const all the things".